sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -21,6 +21,7 @@ import java.net.URI
 import java.util.Date
 
 import com.google.common.base.Objects
+import scala.collection.mutable
```

Comment:
Usually, the import order is:

    import java.util.Date

    import scala.collection.mutable

    import com.google.common.base.Objects

`Dataset.scala` is an example.